python/py-maxminddb/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+#
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                maxminddb
+version             1.5.2
+revision            0
+categories-append   python
+platforms           darwin
+license             Apache Software License (Apache License, Version 2.0)
```

Comment:
`license` is not a free text field and must use approved values. [See the wiki](https://trac.macports.org/wiki/PortfileRecipes#license) for specifics. In this case use:

`license            Apache-2
`